x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -120,11 +129,7 @@ describe('Create Execution Handler', () => {
     mockActionsPlugin.renderActionParameterTemplates.mockImplementation(
       renderActionParameterTemplatesDefault
     );
-    alertExecutionStore = {
-      numberOfTriggeredActions: 0,
-      numberOfGeneratedActions: 0,
-      triggeredActionsStatus: ActionsCompletion.COMPLETE,
-    };
+    ruleRunMetrics = { ...EMPTY_RULE_RUN_METRICS };
```

Comment:
Better name too... i converted this into a class in the other PR though :'(

Reply:
Yes! I like the class conversion so I am waiting for that PR to merge so I can use it :)